executor.go

```diff
@@ -189,7 +189,7 @@ func (e *Executor) Run(ctx context.Context, taskNames []string, runtime *Runtime
 	return e.wg.Wait()
 }
 
-func (e *Executor) shellRun(ctx context.Context, command string, opts ...shell.RunOption) error {
+func (e *Executor) ShellRun(ctx context.Context, command string, opts ...shell.RunOption) error {
```

Comment:
let's godoc this, e.g. why is this different from `shell.Run` - injects defaults for stdout/in for the task and any other executor options

Reply:
Done